digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/dao/RoadAddressDAO.scala

```diff
@@ -529,6 +529,16 @@ object RoadAddressDAO {
       """.as[Long].list
   }
 
+  def getValidRoadNumbersWithFilter = {
```

Comment:
I did not understand/remember why we couldn't use previous method (one with out filter)